presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -151,7 +151,9 @@ public Node visitUse(SqlBaseParser.UseContext context)
     @Override
     public Node visitCreateTableAsSelect(SqlBaseParser.CreateTableAsSelectContext context)
     {
-        return new CreateTableAsSelect(getQualifiedName(context.qualifiedName()), (Query) visit(context.query()), processTableProperties(context.tableProperties()));
+        boolean withData = context.NO() == null;
+        QualifiedName qualifiedName = getQualifiedName(context.qualifiedName());
```

Comment:
I wanted to make to make a return statement to have less code, because there was happening a lot in just a single line.

In-lined.